src/cli/litra-toggle.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+import { program } from 'commander';
+import { toggle } from '../driver';
+import { getDeviceForCLI } from './utils';
+
+program
+  .name('litra-toggle')
+  .description('Toggle a Litra device on or off')
```

Comment:
For consistency with the docs of the other commands:

```suggestion
  .description('Toggles a Litra device on or off')
```